tests/src/Kernel/Paragraphs/FactsFiguresTest.php

```diff
@@ -0,0 +1,193 @@
+    $description_fact = $crawler->filter('div.row-cols-md-3.row > div.col:nth-child(2) > p');
+    $this->assertStringContainsString(
+      'Turpis varius congue venenatis, erat dui feugiat felis.',
+      $description_fact->html()
+    );
+
+    $title_fact = $crawler->filter('div.row-cols-md-3.row > div.col:nth-child(3) > h4.text-capitalize.fw-bold');
+    $this->assertStringContainsString(
+      '107 Tests',
+      $title_fact->html()
+    );
+    $subtitle_fact = $crawler->filter('div.row-cols-md-3.row > div.col:nth-child(3) >  h5.fw-bold');
+    $this->assertStringContainsString(
+      'Test tickets',
+      $subtitle_fact->html()
+    );
+    $description_fact = $crawler->filter('div.row-cols-md-3.row > div.col:nth-child(3) > p');
+    $this->assertStringContainsString(
+      'Cras vestibulum efficitur mi, quis porta tellus rutrum ut. Quisque at pulvinar sem.',
+      $description_fact->html()
+    );
+
+    $title_fact = $crawler->filter('div.row-cols-md-3.row > div.col:nth-child(4) > h4.text-capitalize.fw-bold');
+    $this->assertStringContainsString(
+      '5670 Variants',
+      $title_fact->html()
+    );
+    $subtitle_fact = $crawler->filter('div.row-cols-md-3.row > div.col:nth-child(4) >  h5.fw-bold');
+    $this->assertStringContainsString(
+      'Test variants',
+      $subtitle_fact->html()
+    );
+    $description_fact = $crawler->filter('div.row-cols-md-3.row > div.col:nth-child(4) > p');
+    $this->assertStringContainsString(
+      'Aliquam lacinia diam eu sem malesuada, in interdum ante bibendum.',
+      $description_fact->html()
+    );
+
+    $title_fact = $crawler->filter('div.row-cols-md-3.row > div.col:nth-child(5) > h4.text-capitalize.fw-bold');
+    $this->assertStringContainsString(
+      '345 Dev Ticket',
+      $title_fact->html()
+    );
+    $subtitle_fact = $crawler->filter('div.row-cols-md-3.row > div.col:nth-child(5) >  h5.fw-bold');
+    $this->assertStringContainsString(
+      'Jira ticket',
+      $subtitle_fact->html()
+    );
+    $description_fact = $crawler->filter('div.row-cols-md-3.row > div.col:nth-child(5) > p');
+    $this->assertStringContainsString(
+      'Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. Duis nec lectus tortor.',
+      $description_fact->html()
+    );
+
+    $title_fact = $crawler->filter('div.row-cols-md-3.row > div.col:nth-child(6) > h4.text-capitalize.fw-bold');
+    $this->assertStringContainsString(
+      '43 Components',
+      $title_fact->html()
+    );
+    $subtitle_fact = $crawler->filter('div.row-cols-md-3.row > div.col:nth-child(6) >  h5.fw-bold');
+    $this->assertStringContainsString(
+      'Figma components',
+      $subtitle_fact->html()
+    );
+    $description_fact = $crawler->filter('div.row-cols-md-3.row > div.col:nth-child(6) > p');
+    $this->assertStringContainsString(
+      'Sed efficitur bibendum rutrum. Nunc feugiat congue augue ac consectetur.',
+      $description_fact->html()
+    );
```

Comment:
ok, we asserted the default layout in detail, but now, lets change to 1 column, then 2 columns and for each just assert that the general structure changed to match the columns saved